test/testdata/namer/arguments.rb.flatten-tree-raw.exp

```diff
@@ -77,7 +77,7 @@ InsSeq{
               expr = Local{
                 localVariable = <U b>
               }
-              default_ = EmptyTree
```

Comment:
None of the exp files that change showcase a diff to cfg.exp files.

What do you think about finding some of the tests which change in this diff and recording a `cfg.exp` for them?

Reply:
I added a test that tests some cases where positional and keyword args with defaults interact with the bock arg, and recorded its CFG.